store/json.go

```diff
@@ -25,7 +25,7 @@ const (
 	lockExtension = ".lock"
 
 	// Maximum number of retries before failing a lock call.
-	lockMaxRetries = 200
+	lockMaxRetries = 100
```

Comment:
how quickly do we retry? this could go by very fast. should there be some exponential back-off, maybe?

Reply:
Isn't 10 seconds long enough? 100 retries with 100ms delay

Reply:
I don't know. Usually (like with http calls), we would do an exponential back-off where we wait maybe twice as long every time instead of a fixed interval (ex: 50ms, 100, 200, 400, 800, ...). But if 100x100ms is what we want, that's fine with me